test/integration/install_binary/vault_spec.rb

```diff
@@ -6,7 +6,7 @@
 describe command('/usr/local/bin/vault -version') do
   its(:exit_status) { should eq 0 }
   its(:stderr) { should be_empty }
-  its(:stdout) { should match(/^Vault v0.11.2 \('2b1a4304374712953ff606c6a925bbe90a4e85dd'\)/) }
+  its(:stdout) { should match(/^Vault v1.0.3 \('85909e3373aa743c34a6a0ab59131f61fd9e8e43'\)/) }
```

Comment:
This update actually breaks the manual upgrade test. The test should be against the latest installed version i.e. `v1.1.0`. I'm not sure what value these version changes bring to the testing of the formula anyway.

Reply:
Ah, my bad. This verification is run after the upgrade has completed I take it?